### PR TITLE
[opentelemetry-operator] add annotations and podAnnotations

### PR DESCRIPTION
**Pull request by Sadzeih**

### CODE REVIEW

**Comment by dmitryax** [comment]
@Sadzeih PTAL at the failing CI

**Review by Allex1 — APPROVED** [review]

File: charts/opentelemetry-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -4,6 +4,8 @@ metadata:
   labels:
     app.kubernetes.io/name: opentelemetry-operator
     control-plane: controller-manager
+  annotations:
```
Comment (TylerHelmuth) [inline charts/opentelemetry-operator/templates/deployment.yaml:7]: Lets only add this section if there are values in `.Values.manager.annotations`

**Review by TylerHelmuth — COMMENTED** [review]
Please rebase

File: charts/opentelemetry-operator/examples/default/rendered/admission-webhooks/operator-webhook-with-cert-manager.yaml [hunk 2]
```diff
@@ -14,7 +14,7 @@ webhooks:
     clientConfig:
       service:
         name: opentelemetry-operator-webhook-service
-        namespace: default
+        namespace: monitoring
```
Comment (dmitryax) [inline charts/opentelemetry-operator/examples/default/rendered/admission-webhooks/operator-webhook-with-cert-manager.yaml:17]: Why the namespace changed?

**Review by dmitryax — COMMENTED** [review]
Please rebase

File: charts/opentelemetry-operator/examples/default/rendered/admission-webhooks/operator-webhook-with-cert-manager.yaml (hunk 2 above)
Comment (Sadzeih) [reply charts/opentelemetry-operator/examples/default/rendered/admission-webhooks/operator-webhook-with-cert-manager.yaml]: that was `kubens` playing tricks on me, that should be fixed

**Review by TylerHelmuth — COMMENTED** [review]
@Sadzeih please run `make generate-examples` to re-render the examples with the new chart version.

**Pull request closed without merging**